### PR TITLE
Add Alex to the team

### PR DESCRIPTION
**Pull request by twokul**

### CODE REVIEW

File: source/team.html.erb [hunk 1]
```diff
@@ -86,7 +86,7 @@ responsive: true
 <hr/>
 <section class="team section">
 <h2 class="text-center">
-    Ember Core Alumni
+    Ember Core Emeritus
```
Comment (acorncom) [inline source/team.html.erb]: Hmm, this renames all Ember Core folks from Alumni to Emeritus. Is that what we intend across the board?

Comment (twokul) [reply source/team.html.erb]: Yeah. Based on the conversations with several core team members (last one was Rob), the intention is to call Ember Core folks "emeritus" (preserve all the privileges w/o any further obligations/responsibilities). I'm happy to submit a separate PR if needed

Comment (rwjblue) [reply source/team.html.erb]: Yes, Emeritus is the correct term (and is what we use internally also)

**Comment by twokul** [comment]
@rwjblue @acorncom done and done.

**Pull request merged**